### ISSUE TITLE
italian translation isn't good

### ISSUE THREAD

**Issue by marioforto** [HabitRPG/habitica#4631]
hi, italian translation isn't good, both literally that visually. all boxes are malformed and it's really ugly to see.

it should be:

habits = abitudini
dailies = quotidiane
to dos = da fare

under the section "habits" it should be:

tutte, deboli, forti.

under the section "dailies" it should be:

tutte, non fatte, fatte

under the section "to dos" it should be:

tutte, scadute, fatte.

under the section "rewards" it should be:

tutte, vestiti e abilità

if you visit the italian version you'll notice that all boxes are destroyed, very ugly.

**Comment by crookedneighbor** [comment]
@marioforto I switched my language to Italian to see what you're talking about, in terms of the visual ugliness, but I don't see anything out of the ordinary. Would you mind taking a screenshot? You won't be able to attach it via email, you'll have to log into Github and drag it onto your comment.

For the incorrect translations, would you be interested in translating for us? [Here's a link with more information about translating](https://trello.com/c/SvTsLdRF/12-translations)

**Comment by marioforto** [comment]
the problem comes because i use a zoom of 150% on google chrome

![schermata 2015-02-04 alle 22 55 09](https://cloud.githubusercontent.com/assets/10855531/6050559/f7fd31d0-acc0-11e4-82c1-672948438d39.png)

**Comment by crookedneighbor** [comment]
Ahh, that's because the words are much longer than the English version. If I zoom in to 150% and adjust the width of my screen, I get similar behavior for the equipment tab.

![screen shot 2015-02-04 at 3 59 35 pm](https://cloud.githubusercontent.com/assets/2916945/6050662/087e1310-ac87-11e4-819d-11d935db68b4.png)

We could do some css magic with flexbox (IE 10+) and make it look like this:

![screen shot 2015-02-04 at 4 11 38 pm](https://cloud.githubusercontent.com/assets/2916945/6050865/a4f65c9c-ac88-11e4-9a27-26f042ca97a4.png)

Regarding the actual translations though, @marioforto, are you interested in contributing?

**Comment by marioforto** [comment]
yes that would be a good fix, but words should be changed anyway. i signed up in that site to translate but apparently i can translate only translations that haven't been confirmed twice. i cannot modify most of them

**Comment by crookedneighbor** [comment]
@paglias, what do you think?

**Comment by Alys** [comment]
Also @paglias for this sort of problem, do you prefer for people to raise an issue like this, or comment in Trello on the Translations card? We can add something to the wiki Linguists page to describe your preferred approach.

**Comment by deilann** [comment]
To be a bit blunt here, I'm not sure that we should consider the site not
scaling well with browser zoom to be a "bug." We can't reliably control
what browser zoom does. If browser zoom broke the usability, rather than
the aesthetics of the site, I'd be more inclined to call it a bug, but
trying to make the site entirely zoomable feels to me like trying to
support obscure browsers.

I support making Habit more accessible, but in my own attempts to make the
site more accessible for me, my primary concern has always been\* usability*,
not the "prettiness" of the site. It seems entirely usable... just kind of
ugly, unlike other sites which completely break when browser zoom is used.

I'd also like to point out that most computers have a magnifier that can be
used. I also know many blind folks who use a physical magnifier with their
computer.

tl;dr:

Aesthetics of browser zoom is not something we should consider as a bug.
Usability should be our primary concern, and other tools are available.

**Comment by crookedneighbor** [comment]
@deilann I should clarify, I'm not concerned about browser zoom. I just had it zoomed in on my demo so that the behavior could be replicated. 

I'm more concerned with translations that cause it to overflow without any browser zooming. I'm sure there are languages where there aren't 3 succinct words for the tabs, in those cases, the filter tabs would always be broken.

I'm 1000% for better accessibility, but I also like prettiness :smile:

**Comment by deilann** [comment]
Personally, I would consider this a bug when we run into that language,
where the translator cannot find something succinct in their language that
would convey the feeling (remember: we took a long time in choosing the
words we wanted... there are many alternatives and it doesn't need to be a
direct translation) and not until then.

**Comment by paglias** [comment]
About translations: I'm really behind on them but anyway, transifex indeed
blocks changes to translations reviewed unless a reviewer make the change.
So, I keep habit in English but seeing from the screenshot there are indeed
some problems with the language itself, especially pluralization. The rest
I think are different word choices, maybe not the best one but not incorrect
Il 05/feb/2015 00:09 "deilann" notifications@github.com ha scritto:

**Comment by crookedneighbor** [comment]
@paglias 

Alys asked:

> Also @paglias for this sort of problem, do you prefer for people to raise an issue like this, or comment in Trello on the Translations card? We can add something to the wiki Linguists page to describe your preferred approach.

Do you have a preferred method?

**Comment by paglias** [comment]
They can open issues on individual strings directly on transifex, and all
the reviewers should get an email notification about them so it's probably
the best one
Il 05/feb/2015 00:26 "Blade Barringer" notifications@github.com ha
scritto:

**Comment by Alys** [comment]
Thanks @paglias! I'll add that to the Linguists page in about 30 minutes.

**Comment by marioforto** [comment]
how do i open an issue? should i click "suggestion" or make a comment? 
anyway if you could make me a reviewer i'd simply change the italian strings which sound odd to me

**Comment by marioforto** [comment]
nothing?

**Comment by paglias** [comment]
@marioforto sorry, so to open an issue you should make a comment and tick the "Contrassegna come un problema" (in italian) box, I'll give a look to the suggestions and if everything is ok I'll set you as a reviewer

**Comment by paglias** [comment]
@marioforto ok, I set you as a reviewer. Feel free to correct anything plainly wrong, for the rest i would advise trying to coordinate with the other reviewers if they're still working on it.
